tools/cli/compareBundleAnalysis.js

```diff
@@ -28,10 +28,10 @@ const { argv } = require('yargs')
   .option('baseBranchName', {
     string: true,
     demandOption: true,
-    description: 'Destination branch against which current Pull request\'s bundle size is compared',
+    description: "Destination branch against which current Pull request's bundle size is compared",
   });
 
-const { ADOSizeComparator, getAzureDevopsApi, bundlesContainNoChanges } = require('bundle-size-tools');
+const { ADOSizeComparator, getAzureDevopsApi, bundlesContainNoChanges } = require('../bundle-size-tools');
```

Comment:
Just out of curiosity, why was this change needed?

Reply:
As `yarn` build and store local packages in a `hoisted` form the `compareBundleAnalysis.js` script knows where the package is located. But `pnpm` uses hard links to resolve packages, we need to specify where the exact package is stored in the mono-repo.